Add recent core changes (#600)
owncloud/docs-server: modules/admin_manual/pages/gdpr.adoc

> = GDPR

Reviewer: ```suggestion
= General Data Protection Regulation (GDPR)
```

:toc: right

> :description: When using ownCloud, privacy is the main focus when it comes to data protection. As european company, ownCloud follows the GDPR framework to provide data protection for its customers.

Reviewer: ```suggestion
:description: For ownCloud, privacy is the main focus when it comes to data protection. As a European company, ownCloud follows the GDPR framework to provide data protection for its customers.
```

== Introduction

{description}

== GDPR in ownCloud

> Without claim of completeness, here are several locations in this documentation referencing GDPR.

Reviewer: ```suggestion
GDPR is referenced in several locations throughout this documentation. Some of the more prominent sections are listed below for your convenience:
```
